CONTRIBUTING.md

```diff
@@ -270,6 +270,19 @@ Similar to linting, we recognize documentation can be annoying. If you do not wa
 
 Documentation and the skeleton lives under the `docs/` folder. Example code is imported from under the `examples/` folder.
 
+### To contribute a new documentation notebook:
+
+#### Install dependencies
+
+1. [Quarto](https://quarto.org/) - package that converts Jupyter notebooks (.ipynb files) into mdx files for serving in Docusaurus (only necessary if building docs site locally).
+2. Navigate to the `docs/core_docs/docs/` directory.
+3. Use the provided yarn script to create a new Jupyter notebook:
+```shell
```

Comment:
Why not just create a new file and name it `x.ipynb`?

Reply:
If you want to access anything like `process.` you'll need to import a util file, this script writes a notebook which automatically adds that as the first cell.

Reply:
Can we shim it or start enforcing `getEnvironmentVariable` usage?